lib/ret/storage_used.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Ret.StorageUsed do
+  use Cachex.Warmer
+  use Retry
+
+  def interval, do: :timer.minutes(5)
+
+  def execute(_state) do
+    storage_path = Application.get_env(:ret, Ret.Storage)[:storage_path]
+    
+    case System.cmd("df", ["--output=used", storage_path]) do
```

Comment:
The output flag is a GNU extension, so this is not POSIX compliant, but I guess that doesn't matter if we assume we control the infrastructure.

Reply:
interesting -- this mode is pretty nice since it avoids most of the weird parsing needed, but i'll see if i can do something more compliant. (i'm assuming the habitat coreutils has the GNU variant, which limits some of the risk here insofar as we expect people to use habitat)